examples/gatekeeper/managed-resource/samples/constraint.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: RequireComposition
+metadata:
+  name: crossplane-provider-config-ns
+spec:
+  match:
+    scope: "Cluster"
+    kinds:
+      - apiGroups: ["*"] # not possible to partial match. e.g. *.aws.crossplane.io does not work.
```

Comment:
I am curious, is the lack of a partial match a limitation with gatekeeper?

Reply:
It is a limitation with Gatekeeper.